README.md

```diff
@@ -1,2 +1,70 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
```

Comment:
I've updated this to be clearer that it's for each _type_ of page and given an example as well. https://github.com/everypolitician/scraped_page/pull/1/commits/d62eb3c33da06a1ae7598be1d856680195a11926